swift-ios/Task Tracker/WelcomeViewController.swift

```diff
@@ -150,31 +146,20 @@ class WelcomeViewController: UIViewController {
                     self!.errorLabel.text = "Login failed: \(error!.localizedDescription)"
                     return
                 }
-                
-                guard let user = maybeUser else {
-                    fatalError("Invalid user object?")
-                }
 
                 print("Login succeeded!");
-                
-                if (!USE_PROJECTS_PAGE) {
-    
-                    // For the first phase of the tutorial, go directly to the Tasks page
-                    // for the hardcoded project ID "My Project".
-                    // This will use a common project and demonstrate sync.
-                    let partitionValue = "My Project"
-
-                    // Open a realm.
-                    let projectRealm = try! Realm(configuration: user.configuration(partitionValue: partitionValue))
-
-                    self!.navigationController!.pushViewController(TasksViewController(project: nil, projectRealm: projectRealm), animated: true);
-
-                } else {
-                    // For the second phase of the tutorial, go to the Projects management page. 
-                    // This is where you can manage permissions and collaborators.
-                    self!.navigationController!.pushViewController(ProjectsViewController(), animated: true);
+
+                // Go directly to the Tasks page for the hardcoded project ID "My Project".
+                // This will use a common project and demonstrate sync.
```

Comment:
Nitpick: could you clarify what "this" refers to? Is it a Task? The previous sentence talks about a project ID which I don't think is the subject that "will use a common project and demonstrate sync".

Reply:
Good call. Updated.